code/game/objects/items/explosives/mine.dm

```diff
@@ -293,6 +375,29 @@
 	iff_signal = null
 
 
+// Subtype from this to create buried mines.
+/obj/item/explosive/mine/bury
+	use_tripwire = FALSE
+	needs_digging = TRUE
+	map_prespawn = FALSE
+	buried = FALSE
+	arming_time = 3 SECONDS
+	use_dir = FALSE
+	unacidable = TRUE
+	var/datum/effect_system/spark_spread/sparks = new
```

Comment:
QDEL_NULL on destroy

Reply:
Again, you didn't do the review and didn't give a reason why.

also, codedoc

Reply:
apologies, muddled the way I did the fixes